source-code/core/src/lint/README.md

```diff
@@ -0,0 +1,289 @@
+# `@inlang/core/lint`
+
+This package provides the functionality to automatically check `Resources` for potential errors and other insights.
+
+Use cases could be:
+
+- check if all `Messages` were translated to another language
+- check if the name of a brand always uses the same naming schema e.g. `MyApp` instead of `my-app` or `myApp`
+- check if the tone of a message does not sound aggressive
+- check if the translated message does not exceed a certain length
+- ... and probably a ton of other use cases. Please tell us if we should add one to this list.
+<!-- TODO: link to discussion -->
```

Comment:
do we want to open a discussion for lint rules?

Reply:
Do you refer to `standard-lint-rules` or opening a general discussion for more lint rules in https://github.com/inlang/inlang/discussions? 

If you refer to the latter, sounds reasonable. Opening discussions in `standard-lint-rules` would fragment the discussions too much. I am going to ask GitHub whether they can introduce a feature to let Discussions on repositories all refer to the main discussions.

Reply:
created discussion #406